index.html

```diff
@@ -10,20 +10,29 @@
 </head>
 <body>
     <div class="portfolio">
+        <div class="menuModel" id="model" >
+            <span class="close" id="close" >&times;</span>
+            <ul>
+                <li id="menuItem1" > <a href="#landingPage"> Portfolio </a> </li>
+                <li id="menuItem2" > <a href="#about"> About</a> </li>
+                <li id="menuItem3" > <a href="#contact"> Contact </a> </li>
+            </ul>
+        </div>
         <header>
             <nav class="menuNav" >
                 <a href="#" class="logo">Shayan</a>
                 <ul class="mobileMenu" >
-                    <li><img src="./images/menu.svg" class="menuIcon" alt="menu icon"></li>
+                    <li id="open" ><img src="./images/menu.svg" class="menuIcon" alt="menu icon"></li>
                 </ul>
                 <ul class="menu" >
                     <li>Portfolio</li>
                     <li>About</li>
                     <li>Contact</li>
                 </ul>
+                
             </nav>
```

Comment:
- [ ] Please 😊try to add the grade of the background color of the mobile menu to be like the Figma Design I see😃 that you add ```6070ffd0```, so please try to make it ```6070ff``` without d0 to be like the Figma Design and if you want to blur the background as you did try to add ```mix-blend-mode: multiply``` with the ```backdrop-filter: blur``` which you added it, so with that💯you will reach to the Figma design and mobile menu will be attractive for the visitors😊 

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/215601128-9e52bf9a-2873-4b19-927e-001208ba94e9.png)  |    ![image](https://user-images.githubusercontent.com/83360636/215601045-874a9ccf-12b9-4abe-a071-1824f3a442fa.png)|

Reply:
According to your suggestion, I am having this blending problem
![image](https://user-images.githubusercontent.com/18634639/215610492-897d6e66-106b-4d72-9031-4481fa40be32.png)

Reply:
> According to your suggestion, I am having this blending problem 

- Hi @shayan1234554321 👋This is just a suggestion😃 the importance here is to make it like the Figma Design when previously creating my portfolio like you now, I used the mix-blend and it is working for me, so just I suggest for you 😊

Reply:
Okay, Thanks for the great review. I did learn much from your review 🥰.